src/main/java/io/cryostat/net/web/WebModule.java

```diff
@@ -61,7 +61,6 @@ public abstract class WebModule {
     public static final String WEBSERVER_TEMP_DIR_PATH = "WEBSERVER_TEMP_DIR_PATH";
 
     @Provides
-    @Singleton
```

Comment:
From the Vert.x Core docs on Worker Verticles: 

"These run using a thread from the worker pool. An instance is never executed concurrently by more than one thread."

Is this why we no longer need to ensure the `WebServer` is a singleton (i.e. the WebServer Worker Verticle instance will only ever be executed on one thread)?

Reply:
In practice there is still only one WebServer instance since `Cryostat.java` only asks for one instance. But, it might be useful in the future to also be able to deploy mutliple WebServer verticles, since that's also a recommended way to structure concurrency with Vertx.